src/Command/ModuleRefresh.php

```diff
@@ -58,7 +58,12 @@ public function execute(InputInterface $input, OutputInterface $output)
     {
         try {
             $this->logger->info('Refreshing modules started.');
-            $this->config->refresh();
+            $enabledModules = $this->config->refresh();
+            $this->logger->info(
+                $enabledModules ?
+                    'The following modules have been enabled:' . PHP_EOL . implode(PHP_EOL, $enabledModules) :
+                    'No modules were changed.'
+            );
```

Comment:
This code is duplicated in src/Process/Build/RefreshModules.php. We should refactor it into src/Config/Module.php

Reply:
It's not a responsibility of `src/Config/Module.php` class to write into log files.
I'll refactor it by using RefreshModules process in ModuleRefresh command.

Reply:
I'm fine with that solution.